### PR TITLE
Me and Ayo planned the Ads process and created the model

### PR DESCRIPTION
**Pull request by y-adeleke**
Please review what we did!

### CODE REVIEW

**Review by binidowu — COMMENTED** [review]
😀😀

**Comment by y-adeleke** [comment]
Guy mistake dey that code ooo but it’s minor.
In the ads model…there are two Schema which should be mongoose.Schema

Yusuf O. Adeleke

File: web9ja_backend/models/ads.js [hunk 1]
```diff
@@ -1,34 +1,146 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
+
+// Question Schema
+const questionSchema = new mongoose.Schema({
+  questionText: { type: String, required: true },
+  askedBy: { type: Schema.Types.ObjectId, ref: "User", required: false },
```
Comment (binidowu) [inline web9ja_backend/models/ads.js:6]: askedBy: { type: mongoose.Schema.Types.ObjectId, ref: "User", required: false },

File: web9ja_backend/models/ads.js [hunk 2]
```diff
@@ -1,34 +1,146 @@
...
-        type: Date,
-        default: Date.now
-    },
-
-    publisher: {
-        type: mongoose.Schema.Types.ObjectId,
-        ref: 'User',
-        required: true
+  userId: {
+    type: Schema.Types.ObjectId,
```
Comment (binidowu) [inline web9ja_backend/models/ads.js:13]: type: mongoose.Schema.Types.ObjectId

**Review by binidowu — COMMENTED** [review]
😀

File: web9ja_backend/models/ads.js (hunk 2 above)
Comment (Abdul4real) [reply web9ja_backend/models/ads.js:13]: hmm

**Comment by Abdul4real** [comment]
Thumbs up!

**Review by Abdul4real — APPROVED** [review]

**Pull request merged**